### ISSUE TITLE
google-chrome 69: private sessions looks almost the same than standard

### ISSUE THREAD

**Issue by aslafy-z** [NicoHood/arc-theme#142]
### Details

- Distribution: Archlinux
- Desktop environment: i3-gaps 4.15
- Arc version: community/arc-gtk-theme/20180715-1
- GTK+ 3 version:
  - Murrine engine version: 0.98.2-3
  - `gnome-themes-extras` version: N/A (running i3)
- Installation method: archlinux package 

![2018-10-09-151304_1920x1080_scrot](https://user-images.githubusercontent.com/8191198/46671757-d9f53b00-cbd5-11e8-8b14-805dc042a539.png)

**Comment by jnsh** [comment]
Do you actually have some idea of how to make this better?

The Arc-Dark(er) is dark by design and that can't really be changed, and the chrome/chromium private window is dark by google's design. Even if it was possible to change the private window theming in the gtk3 theme (it may or may not be, I'm really not sure), I can't think of any sensible way to improve the design.

**Comment by aslafy-z** [comment]
No I didn't, I'm gonna use the old 'Hybrid' UI

How: https://askubuntu.com/questions/1072234/how-to-get-back-my-previous-omnibox-in-google-chrome-69-or-chromium-69

Thanks @jnsh https://github.com/NicoHood/arc-theme/issues/141#issuecomment-428250906

**Comment by jnsh** [comment]
Is there something different with the "Hybrid UI" that makes private browsing windows more distinguishable?

The thing is, that unless you have some specific suggestions, I don't really see much that could be done to this. I think the same issue affects every other dark theme as well, including Adwaita-Dark, and every other browser that chooses to use the dark theme with the private/incognito windows, e.g. Firefox.

It'd be better bug the browser developers about this, and now that there's at least one "mainstream" OS, that provides a dark theme (Mac OS), the issue might get some more attention as well.

**Comment by aslafy-z** [comment]
![2018-10-09-204243_1920x1080_scrot](https://user-images.githubusercontent.com/8191198/46690940-ea240f00-cc03-11e8-90c6-ed166d0cf987.png)

There is a little icon that make the difference, I assume that google-chrome 69 ditched it since it does not appear when I switch back to default UI and Bundled theme instead of GTK+ one.

**Comment by jnsh** [comment]
Yes, apparently the incognito icon was moved to the left next to the address bar.

One thing you could do, is force chrome to use the light theme variant with `GTK_THEME` environment variable [documented in this page](https://developer.gnome.org/gtk3/stable/gtk-running.html), although I understand that may not be preferable solution at all.
